src/test/groovy/io/codearte/jfairy/producer/person/locale/de/DeAddressSpec.groovy

```diff
@@ -0,0 +1,60 @@
+package io.codearte.jfairy.producer.person.locale.de
+
+import io.codearte.jfairy.Fairy
+import io.codearte.jfairy.producer.person.Address
+import spock.lang.Specification
+
+import static org.apache.commons.lang3.SystemUtils.LINE_SEPARATOR
+
+class DeAddressSpec extends Specification {
+
+	private final int SEED = 7
+	private Fairy fairy;
+	private Address address
+
+	def setup() {
+		fairy = Fairy.builder().withRandom(new Random(SEED)).withLocale(Locale.GERMAN).build()
+		address = fairy.person().address
+	}
+
+	def "should generate random street"() {
+		expect:
+			address.street == "Messelweg"
```

Comment:
Please convert to Groovy String in place of GString. Please do so for all the GStrings where a Groovy String could be used instead.